rcl_yaml_param_parser/Doxyfile

```diff
@@ -0,0 +1,29 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl_yaml_param_parser"
+PROJECT_NUMBER         = master
+PROJECT_BRIEF          = "Package containing various utility types and functions for C."
+
+INPUT                  = ./include
```

Comment:
This repo does have a README.md, so you could use that as the landing page of its docs.